### PR TITLE
add link to collaborative workflows slides

### PR DESCRIPTION
**Pull request by munkm**
My slides should render nicely! I'm very excited about them.

### CODE REVIEW

**Comment by jpmorgan98** [comment]
wow this is dope!

**Review by kyleniemeyer — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Please show the [guest pass](./Z5039%20--%20URSSI%20Software%20School.png) on yo
 |:--|:--|:--|:--|
 | 3rd, 1-2pm  | [Welcome and introductions](https://kyleniemeyer.github.io/research-software-dev-modules/intro-slides) |  | Kyle
 | 3rd, 2-5pm  | [Software design and modularity](https://evamaxfield.github.io/winter-school-lectures/software-design-and-modularity.slides.html#/),<br/>[Structuring Python packages](https://kyleniemeyer.github.io/research-software-dev-modules/module-packaging/) |  | Eva, Kyle
-| 4th, 9am-12pm | [Collaboration with Git/GitHub/Workflows](), [Git Exercises](), work time |  | Madicken
+| 4th, 9am-12pm | [Collaboration with Git/GitHub/Workflows](https://munkm.github.io/2024-winterschool/git-collaboration), [Git Exercises](), work time |  | Madicken
```
Comment (kyleniemeyer) [inline README.md]: there seems to be a typo, I retract my approval

**Comment by cassidymwagner** [comment]
I can't see it ):

**Review by kyleniemeyer — CHANGES_REQUESTED** [review]
Please correct the typo 😄

**Review by jpmorgan98 — COMMENTED** [review]
I think you made a typo ion the url!

**Comment by munkm** [comment]
Oh no! I'm so sorry! I forgot to link to the HTML. I'll fix it!

**Comment by elotech47** [comment]
Link works now !!!

**Review by kyleniemeyer — APPROVED** [review]
Looks correct now

**Review by jpmorgan98 — APPROVED** [review]
This looks like it works!

**Comment by munkm** [comment]
Ok, I think I've fixed it. I'm excited to contribute to this school!!!

**Pull request merged**